tests/test_deployment.py

```diff
@@ -457,6 +457,15 @@ def test_scales_dynos(self, dsss, heroku_mock, active_config):
             ]
         )
 
+    def test_scales_different_dynos(self, dsss, heroku_mock, active_config):
+        active_config.set("dyno_type", u"ignored")
+        active_config.set("dyno_type_web", u"tiny")
+        active_config.set("dyno_type_worker", u"massive")
+        dsss(log=mock.Mock())
+        heroku_mock.scale_up_dyno.assert_has_calls(
+            [mock.call("web", 1, u"tiny"), mock.call("worker", 1, u"massive"),]
```

Comment:
Looks like `black` set you up for a scolding by `flake8`(?)

Reply:
Ugh, that's exactly what happened, I wrote it flake8 compatible and then black put it on a single line.